### ISSUE TITLE
not worked

### ISSUE THREAD

**Issue by ghasminia** [milgradesec/ratelimit#108]
I am using it on Docker, it does not limit requests
This is how I request:
for i in {1..10}; do dig @myip mydomain; done
Rate limit is set to 1
Responds to all requests

Is this test method correct?

**Comment by milgradesec** [comment]
Can you verify the plugin is actually builded into CoreDNS and running?? 

You can use the `coredns_plugin_enabled` Prometheus metric from https://coredns.io/plugins/metrics/

**Comment by milgradesec** [comment]
Also, ratelimit only applies to UDP requests

**Comment by ghasminia** [comment]
> Can you verify the plugin is actually builded into CoreDNS and running??

Yes, I have access to the ratelimit tag in the corefile on the image I made, there was an error before.
There is also Mirem in the plugins in the container
